src/Tizen.NUI/src/public/Utility/Palette.cs

```diff
@@ -584,7 +570,6 @@ public Tizen.NUI.Color GetBodyTextColor()
             /// return the number of pixels represented by this swatch.
```

Comment:
return -> Returns, and I'm not sure about "represented by" - who is representing who? maybe "pixels that belong to this swatch", "pixels that has been assigned to this swatch", "pixels matching this swatch" - it depends on how you would describe the process of deciding if a pixel is related to a swatch.